js/packages/core/src/open-eyes.ts

```diff
@@ -80,6 +80,8 @@ export function makeOpenEyes<TDriver, TContext, TElement, TSelector, TDefaultTyp
           testConcurrency: concurrency,
           concurrentRendersPerTest: (settings as OpenSettings<'ufg'>).renderConcurrency,
           node: {version: process.version, platform: process.platform, arch: process.arch},
+          driverUrl: spec?.extractHostName?.(target),
```

Comment:
It is also worth checking for the target is not undefined
```suggestion
          driverUrl: target && spec?.extractHostName?.(target),
```